prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -361,6 +377,81 @@ ssize_t rxr_pkt_post_ctrl_or_queue(struct rxr_ep *ep, int entry_type, void *x_en
 	return err;
 }
 
+/*
+ * This function should be called before sending a message with the new protocol
+ * for FI_DELIVERY_COMPLETE.
```

Comment:
This function is used for any extra feature that does not have an alternative. FI_DELIVERY_COMPLETE is an example.

Reply:
" This function should be called before sending a message with the new protocol for FI_DELIVERY_COMPLETE." does not sound right. 

may be something like: "This function is used for any extra feature that does not have an alternative, for example FI_DELIVERY_COMPLETE"